### PR TITLE
Upload replace fix

### PR DESCRIPTION
**Pull request by blakeburch**
- Fix logic so that "replace" method first replaces, then appends.
- Updated bit.io to abide by new SSL requirements
- Decrease definition of "Small file size" to 50mb (to avoid potential memory issues)

### CODE REVIEW

File: postgresql_blueprints/upload_file.py [hunk 1]
```diff
@@ -128,58 +128,49 @@ def combine_folder_and_file_name(folder_name, file_name):
     return combined_name
 
 
+def determine_upload_method(db_connection):
+    if 'db.bit.io' in str(db_connection):
+        upload_method = bitio_upload_method
```
Comment (ericelsken) [inline postgresql_blueprints/upload_file.py:133]: bitio_upload_method is a function? Why are you setting that to a variable?

File: postgresql_blueprints/upload_file.py [hunk 2]
```diff
@@ -128,58 +128,49 @@ def combine_folder_and_file_name(folder_name, file_name):
...
+                # append to the end.
+                insert_method = 'append'
+
             chunk.to_sql(
                 table_name,
                 con=db_connection,
                 index=False,
                 if_exists=insert_method,
-                method='multi',
+                method=upload_method,
```
Comment (ericelsken) [inline postgresql_blueprints/upload_file.py:173]: Still using upload_method and not insert_method. Is that correct?

**Review by ericelsken — CHANGES_REQUESTED** [review]

File: postgresql_blueprints/upload_file.py (hunk 2 above)
Comment (blakeburch) [reply postgresql_blueprints/upload_file.py:173]: Yes. Insert_method is `append` or `replace` and indicates how the data should be added to the table. Upload method is `multi` or `bitio_upload_method` and indicates how the process should be performed (multiple insert statements or COPY from stdin).

File: postgresql_blueprints/upload_file.py (hunk 1 above)
Comment (blakeburch) [reply postgresql_blueprints/upload_file.py:133]: To be honest, I'm not sure on the specifics. That method was directly pulled from bit.io documentation and they call it without the `()`.

https://docs.bit.io/docs/connecting-via-pandas#loading-dataframes-quickly-to-bitio

**Review by ericelsken — APPROVED** [review]

**Pull request merged**